tests/yam/agama/boot_agama.pm

```diff
@@ -39,6 +39,11 @@ sub prepare_boot_params {
         return @params;
     }
 
+    if (get_var('INIT_DUD')) {
+        my $init_dud = get_var('INIT_DUD');
+        push @params, 'inst.dud=' . data_url($init_dud) . ' rd.neednet=1';
```

Comment:
could you do this block in one line?
` push @params, 'inst.dud=' . data_url(get_var('INST_DUD')) . ' rd.neednet=1' if get_var('INST_DUD');`

Reply:
updated.